pennylane/ops/op_math/condition.py

```diff
@@ -52,11 +52,19 @@ class Conditional(Operation):
     def __init__(self, expr, then_op: Type[Operation], id=None):
         self.meas_val = expr
         self.then_op = then_op
-        super().__init__(wires=then_op.wires, id=id)
+        super().__init__(*then_op.data, wires=then_op.wires, id=id)

```

Comment:
I'm wondering whether this change has far-reaching consequences? :thinking: 
Should we mention this as a breaking change or is it purely technical?

Reply:
Yup, makes sense!